starter-code/index.js

```diff
@@ -1,10 +1,89 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+let SortedList = function() {
+    this.items = [];
+    this.length = this.items.length;
+};
+
+SortedList.prototype.add = function(item) {
+    
+    this.items.push(item)
+    this.items.sort(function(i1, i2){
+        return i1-i2});
+    
+    this.length++;
+}
+SortedList.prototype.get = function(pos) {
+    if(pos > this.items.length){
+        return this.OutOfBounds();
+    }
+    return this.items[pos-1];
+}
+SortedList.prototype.max = function() {
+    if (this.items === 0) {
+        return this.EmptyList();
+    } else {
+        var max = this.items[0];
+        this.items.forEach(function(e) {
+            if(e > max){
+                max = e;
+            }
+        }.bind(this));
+    }
+    return max; 
```

Comment:
El `forEach` no es necesario ni la comprobación que hacéis dentro, recordad que habéis hecho un `.sort()` antes. Eso ordena vuestro array, de tal manera que según cómo lo hayáis ordenado el más grande siempre estará al principio o al final.